libdevcore/CommonData.h

```diff
@@ -166,6 +195,68 @@ inline std::string formatNumber(u256 const& _value)
 		return _value.str();
 }
 
+/// @a T will typically by unsigned, u160, u256 or bigint.
+template <class T>
+inline std::string formatNumberReadable(
```

Comment:
This function would sit better in `StringUtils.h`

Reply:
Hex could stay there in my opinion (it's about data).
`StringUtils` has other human readable functions, so I think only `formatNumberReadable` should go there.

Reply:
Okay... I'll rearrange now